store/view/articles.js

```diff
@@ -50,11 +50,12 @@ export const state = () => ({
 // Module actions
 export const actions = {
   getMainPage: composer.compose({
+    name: '#getMainpage',
```

Comment:
const, and should be "#getMainPage" as stated before main page is two separate words